src/lib/utils/menuItems.js

```diff
@@ -1,24 +1,37 @@
 import { IconCar } from '@tabler/icons-svelte';
 
+export const SectionName = {
+  HERO: 'Hero',
+  MEDIA: 'Media',
+  CONTRIBUTORS: 'Contributors',
+  CONTACT: 'Contact',
+  ABOUT: 'About',
+}
+
 export const menuItems = [
   {
-    ref: '#section1',
+    ref: `#${SectionName.HERO}`,
     name: 'Welcome',
     icon: IconCar
   },
   {
-    ref: '#section2',
-    name: 'Media',
+    ref: `#${SectionName.ABOUT}`,
+    name: 'About',
     icon: IconCar
   },
   {
-    ref: '#section3',
-    name: 'Reach Out',
+    ref: `#${SectionName.MEDIA}`,
+    name: 'Media',
     icon: IconCar
```

Comment:
I also noticed that every item as the same icon `IconCar`

Ive [created an issue](https://github.com/Samoa-Stack-Overflow/sso-website/issues/49) for this to be addressed if need be. 

For now I won't bloat the PR even more than it needs to be